src/components/common/PageContainer.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+  <div class="tm-page">
+    <TmPageHeader v-if="!hideHeader" :tabs="tabs">
+      <h2 v-if="title" slot="title">
+        {{ title }}
+      </h2>
+      <h3 v-if="subtitle" slot="subtitle">
+        {{ subtitle }}
+      </h3>
+      <slot slot="menu-body" name="menu-body">
+        <TmBalance v-if="session.signedIn" />
+      </slot>
+      <slot slot="header-buttons" name="header-buttons" />
+    </TmPageHeader>
+    <main class="tm-page-main">
+      <slot name="managed-body" />
+    </main>
+    <PageFooter />
+  </div>
+</template>
+
+<script>
+import TmPageHeader from "./TmPageHeader.vue"
+import { mapState, mapGetters } from "vuex"
+import TmBalance from "common/TmBalance"
+import PageFooter from "common/TmPageFooter"
+
+export default {
+  name: `tm-page`,
```

Comment:
```suggestion
  name: `page`,
```

Reply:
Might make it `page-container` for consistency